### PR TITLE
CanvasJs Trace Added

### PR DESCRIPTION
**Pull request by dhavalveera**
### Technology Request

**Examples:**

**using `script`:**
https://wow-petguide.com/

**using `js`:**
https://www.globalpropertyguide.com/
https://app.guardian.periscopedata.com/login?controller=welcome
https://fcx.com/
https://orientation.georgian.edu/
https://www.policyed.org/

### CODE REVIEW

File: src/technologies/c.json [hunk 1]
```diff
@@ -529,6 +529,19 @@
     "saas": true,
     "website": "https://www.instructure.com/canvas"
   },
+  "CanvasJs": {
```
Comment (nurbek91) [inline src/technologies/c.json:532]: CanvasJS

File: src/technologies/c.json [hunk 2]
```diff
@@ -529,6 +529,19 @@
...
+    "cats": [
+      25
+    ],
+    "description": "CanvasJS charts is a data visualisation library that runs across multiple devices and browsers.",
+    "icon": "canvasjs.png",
```
Comment (nurbek91) [inline src/technologies/c.json:537]: CanvasJS svg

File: src/technologies/c.json [hunk 3]
```diff
@@ -529,6 +529,19 @@
...
+    "js": {
+      "CanvasJS": "",
+      "CanvasJS.Chart": ""
+    },
+    "scriptSrc": "/.+\\.canvasjs/.+\\.js(?:\\?v=(\\d+(?:\\.\\d+)+))?\\;version:\\1",
```
Comment (nurbek91) [inline src/technologies/c.json:542]: How will this code work in such cases?
http://www.tazabek.kg/
/ui-js/admingraph/canvasjs-1.4.1/canvasjs.min.js

https://www.wow-petguide.com/js/jquery.canvasjs.min.js?v=1.52
https://www.architectsjournal.co.uk/wp-content/plugins/aj100/js/jquery.canvasjs.min.js?ver=1.3

CanvasJS.Chart.version return Chart version btw

**Review by nurbek91 — CHANGES_REQUESTED** [review]
Hey mate, thanks for PR. Need improvements

**Review by nurbek91 — CHANGES_REQUESTED** [review]
according previous request

**Comment by dhavalveera** [comment]
> according previous request

now directly checking the `/canvasjs.min` part

**Comment by nurbek91** [comment]
image

**Comment by nurbek91** [comment]
Open PR back please. I'll post in the comments later today how to fix it

**Comment by nurbek91** [comment]
```
 "CanvasJS": {
    "cats": [
      25
    ],
    "description": "CanvasJS charts is a data visualisation library that runs across multiple devices and browsers.",
    "js": {
      "CanvasJS.Chart": "",
      "CanvasJS.Chart.version": "^v(.+)$\\;version:\\1"
    },
    "website": "https://canvasjs.com"
```

File: src/technologies/c.json [hunk 4]
```diff
@@ -529,6 +529,19 @@
...
+    "cats": [
+      25
+    ],
+    "description": "CanvasJS charts is a data visualisation library that runs across multiple devices and browsers.",
+    "icon": "canvasjs.svg",
+    "js": {
+      "CanvasJS.Chart": "",
+      "CanvasJS.Chart.version": "^v(.+)$\\;version:\\1"
+    },
+    "scriptSrc": "/canvasjs/.+\\.js(?:\\?v=(\\d+(?:\\.\\d+)+))?\\;version:\\1",
```
Comment (nurbek91) [inline src/technologies/c.json:542]: delete this one

**Review by nurbek91 — APPROVED** [review]

**Pull request merged**